mhctools/netmhcii_pan.py

```diff
@@ -65,16 +78,12 @@ def prepare_allele_name(self, allele_name):
                     allele.species,
                     allele.gene,
                     allele.allele_code)
-            else:
-                # assume that we're dealing with a human DRB allele
-                # which NetMHCIIpan treats differently because there is
-                # supposedly no population diversity in the DR-alpha gene
-                return "%s_%s%s" % (
-                    allele.gene,
-                    allele.allele_family,
-                    allele.allele_code)
+            return self._prepare_drb_allele_name(allele)
```

Comment:
why factor this out into its own function but not the previous few lines? seems better to be able to see them next to each other. maybe make separate functions for all the different formats?